### DISCUSSION TITLE
Add Pandora support via Pianobar

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#1138] [Ideas]
It would be nice to have support for pianobar. Three are many web gui's out there but it would be nice to have it built into myMPD. I know it's not MPD related, but it would be convenient to have all music needs in one place.

https://github.com/PromyLOPh/pianobar

**Comment by jcorporation** [comment]
`Pandora isn't available in this country right now...`. I can not even see what this is exactly, but anyway it is unlikely that I implement features that have nothing to do with mpd.

myMPD is focused to become the best mpd client in the universe :D
